samples/springboot3/pet-store/pom.xml

```diff
@@ -55,7 +55,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-shade-plugin</artifactId>
-                        <version>3.5.1</version>
+                        <version>3.5.0</version>
```

Comment:
No need to downgrade or were you facing any issues with the latest version?

Reply:
>3.5.0  Was being marked as not found, so I downgraded to the suggested version.
`Plugin 'org.apache.maven.plugins:maven-shade-plugin:3.5.1' not found`

Reply:
I was able to build just fine though, but the warning does not go away.

Reply:
Maybe related to your local cache? It's available in central: https://repo1.maven.org/maven2/org/apache/maven/plugins/maven-shade-plugin/3.5.1/
Can you please remove the change from the PR? I will merge afterwards. Thanks.